lib/hello_web/components/weather.ex

```diff
@@ -0,0 +1,14 @@
+defmodule HelloWeb.Weather do
+  use Phoenix.Component
```

Comment:
@YanethM what does a `Phoenix.Component` do?

Reply:
A component is a function that can be called inside heex and can be in the same module or an external one, as in this case, I call the component in the show.html.heex view.